src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/basic/GT_MetaTileEntity_CropHarvestor.java

```diff
@@ -421,7 +421,7 @@ public boolean applyFertilizer(ICropTile aCrop) {
 			return false;
 		} else {
 			//Logger.INFO("Current Nutrient: "+aCrop.getNutrientStorage()+" for "+aCrop.getCrop().displayName());
-			aCrop.setNutrientStorage(100);
+			aCrop.setNutrientStorage(getNutrientStorage() + 100);
```

Comment:
I think you forgot something
```suggestion
			aCrop.setNutrientStorage(aCrop.getNutrientStorage() + 100);
```